test_runner/regress/test_neon_cli.py

```diff
@@ -131,3 +135,54 @@ def test_cli_start_stop(neon_env_builder: NeonEnvBuilder):
     # Default stop
     res = env.neon_cli.raw_cli(["stop"])
     res.check_returncode()
+
+
+def test_parse_project_git_version_output_positive():
```

Comment:
Because we parametrize all the tests by Build Type and Postgres version, and because these 3 tests don't depend on those parameters, I'd run them only for one Postgres version and one Build Type.


```python
@skip_on_postgres(PgVersion.V14, reason="...")
@pytest.mark.skipif(os.environ.get("BUILD_TYPE") == "debug", reason="...") 
```